Program.cs

```diff
@@ -0,0 +1,410 @@
+﻿using System;
+using VisualCode.Entities;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace VisualCode
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            List<Customers> customersList = new List<Customers>();
+            List<Product> productList = new List<Product>();
+            List<Seller> sellerList = new List<Seller>();
+            List<Invoice> invoiceList = new List<Invoice>();
+            List<InvoiceDetail> invoiceDetailList = new List<InvoiceDetail>();
+
+            LoadDataDefault();
+
+            MenuPrincipal();
+            ProcesoPrincipal();
+
+            //Console.WriteLine($"escribiste {linea}");
+
+            void ProcesoPrincipal()
+            {
+                string opcion = Console.ReadLine();
+
+                switch (opcion)
+                {
+                    case "1":
+                        MenuMaintenance();
+
+                        opcion = Console.ReadLine();
+
+                        switch (opcion)
+                        {
+                            case "1":
+                                RegistryCustomer();
+                                break;
+                            case "2":
+                                RegistryProduct();
+                                break;
+                            case "3":
+                                RegistrySeller();
+                                break;
+                            case "9":
+                                MenuPrincipal();
+                                ProcesoPrincipal();
+                                break;
+                        }

```

Comment:
What could happen if I write 10 as option? try always to use the default keyword to avoid an unexpected situation